tests/test_flow/test_persistence_compatibility.py

```diff
@@ -25,12 +26,18 @@ def older_serialized_cache_harness(make_counter, tmp_path):
 # To renegerate cache, run the following command from bionic/ dir
 #   `python -m tests.test_flow.generate_test_compatibility_cache`
 def test_caching_compatibility(older_serialized_cache_harness):
-    flow = older_serialized_cache_harness.flow
-    assert flow.get("xy") == 6
-    assert flow.get("yz") == 12
-    assert flow.get("xy_plus_yz") == 18
+    # Auto versioned flow requires Python 3.7 or higher.
+    if sys.version_info < (3, 7):
+        flows = [older_serialized_cache_harness.manual_flow]
```

Comment:
wondering, why not just test both manual and auto flows?

Reply:
Mostly for the sake of parity. I tried parameterize the test based on the value of a fixture but unfortunately, pytest doesn't allow for dynamic parameterization.